orchestration/dagster_orchestration/hca_orchestration/solids/load_hca/data_files/load_data_metadata_files.py

```diff
@@ -117,11 +117,9 @@ def ingest_metadata_for_file_type(
     return file_metadata_fanout_result
 
 
-@composite_solid
```

Comment:
I've simplified things in this branch of the pipeline to eliminate the anti-pattern of a composite calling another composite.